roles/nginx/files/fastcgi_cache.conf

```diff
@@ -0,0 +1,19 @@
+set $skip_cache 0;
+
+# POST requests and urls with a query string should always go to PHP
+if ($request_method = POST) {
```

Comment:
@austinpray You are totally right, in fact it is not even used as Nginx default cached method are only GET and HEAD. http://nginx.org/en/docs/http/ngx_http_fastcgi_module.html#fastcgi_cache_methods

I updated accordingly.